packages/components/stories/icon/icons.js

```diff
@@ -1,8 +1,7 @@
 // auto-generated file. DO NOT MODIFY.
 
-export const icons =
-{
-    'appstore': [
+export const icons = {
```

Comment:
This formatting should not be affected. Please revert formatting changes.

Reply:
all changes in icon, js files are happened after doing build on local machine. I added new icons and that’s why I made a build.